Mobile/starter_project_dartsmiths/lib/features/example/presentation/widgets/postCard.dart

```diff
@@ -0,0 +1,123 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class PostCard extends StatelessWidget {
+  final String img;
+  final String topic;
+  final String title;
+  final String likes;
+  final String time;
+  final bool saved;
+  const PostCard(
+      {super.key,
+      required this.img,
+      required this.topic,
+      required this.title,
+      required this.likes,
+      required this.time,
+      this.saved = false});
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 10),
+      child: Container(
+        height: 140,
+        width: MediaQuery.of(context).size.width,
+        decoration: BoxDecoration(
+          borderRadius: BorderRadius.circular(13),
+          color: Colors.white,
+          boxShadow: [
+            BoxShadow(
+              color: Color.fromARGB(255, 240, 240, 240),
+              spreadRadius: 1,
+              blurRadius: 15,
+              offset: const Offset(0, 15),
+            ),
+          ],
+        ),
+        child: Row(
+          children: [
+            Container(
+              // height: 140,
+              width: 100,
+              decoration: BoxDecoration(
+                  borderRadius: BorderRadius.circular(13),
+                  image: DecorationImage(
+                      fit: BoxFit.cover, image: NetworkImage(img))),
+            ),
+            Container(
+              width: 230,
+              child: Padding(
+                padding: const EdgeInsets.all(20.0),
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    Text(topic,
+                        style: GoogleFonts.poppins(
+                          color: Color.fromARGB(255, 55, 107, 237),
+                          fontSize: 16,
+                          fontWeight: FontWeight.w200,
+                          fontStyle: FontStyle.italic,
+                        )),
+                    Text(title,
+                        maxLines: 2,
+                        overflow: TextOverflow.ellipsis,
+                        softWrap: false,
+                        style: GoogleFonts.poppins(
+                          fontSize: 14,
+                          fontWeight: FontWeight.w400,
+                        )),
+                    // SizedBox(
```

Comment:
[nit] better not to have comments in a mergable code
if you don't need it, get rid of it

Reply:
you can search for your comments and get rid of them all if they are just unused code 
just reserve the ones that are documenting what something is